src/components/ActionsMenu/Item/ActionsMenuItem.st.css

```diff
@@ -0,0 +1,97 @@
+:import {
+    -st-from: "../../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+    -st-from: "../../Text/Text.st.css";
+    -st-default: Text;
+}
+
+:vars {
+    /*Defaults*/
+    ItemFontDesktop: "{theme: 'Body-M', size: '16px', lineHeight: '24px'}";
+    ItemSubtitleFontDesktop: "{theme: 'Body-M', size: '14px', lineHeight: '20px'}";
+    ItemFontMobile: "{theme: 'Body-M', size: '14px', lineHeight: '24px'}";
+    ItemSubtitleFontMobile: "{theme: 'Body-M', size: '12px', lineHeight: '20px'}";
+}
+
+:vars {
+    /*Overrides*/
+    ItemFont:  --overridable;
+    ItemSubtitleFont:  --overridable;
+}
+
+.root {
+    -st-states: disabled, mobile;
+    height: 40px;
+    padding: 8px 24px;
+    display: flex;
+    cursor: pointer;
+    position: relative;
+}
```

Comment:
Take a look at this snapshot I took:
![image](https://user-images.githubusercontent.com/3277080/65326121-661b2b00-dbb9-11e9-953d-dcfbbd437591.png)
Note that you can see the main container's border.
And we need the hover background to take all the width (including the border).
So we need to have the item take `width: calc(100% + value(parentsBorder));`
And for that to work we also need to add `box-sizing: border-box` to the ActionsMenu .root